LifeCycleAssessment_Engine/Compute/LCAToolPart.cs

```diff
@@ -0,0 +1,869 @@
+
+            double am52_syst = n27_heating / 1000.0;                //heating energy
+            double aj51_syst = 0.95;                                //convector(heater) Existing Buildings (efficiency)
+            double ag52_syst = am52_syst / aj51_syst;               //convector(heater) Existing Buildings
+
+            double ae54_syst = 0.5;                                 //heat pump heizen Existing Buildings (efficiency)
+            double ae51_syst = 40.0;                                  //Tc
+            double ae52_syst = 50.0;                                  //Th
+            double ae53_syst = (ae52_syst + kelvin) / ((ae52_syst + kelvin) - (ae51_syst + kelvin));
+            double ae55_syst = ae54_syst * ae53_syst;               //COPreal
+            double ae57_syst = ag52_syst / ae55_syst;               //heat pump heating Existing Buildings (power)
+
+            double ab52_syst = ag52_syst - ae57_syst;               //heat pump heating Existing Buildings 
+
+            double am61_syst = n27_hotwater / 1000.0;               //hot water power Existing Buildings
+            double aj60_syst = 0.95;                                //hot water system Existing Buildings (efficiency)
+            double ag61_syst = am61_syst / aj60_syst;               //hot water system Existing Buildings
+            double ae63_syst = 0.5;                                 //heat pump hot water Existing Buildings (efficiency)
+            double ae60_syst = 40.0;                                  //Tc
+            double ae61_syst = 70.0;                                  //Th
+            double ae62_syst = (ae61_syst + kelvin) / ((ae61_syst + kelvin) - (ae60_syst + kelvin)); //COPideal
+            double ae64_syst = ae63_syst * ae62_syst;               //heat pump hot water Existing Buildings COPreal
+            double ae66_syst = ag61_syst / ae64_syst;               //heat pump hot water Existing Buildings (power)
+            double ab61_syst = ag61_syst - ae66_syst;               //hot water system Existing Buildings
+
+            double x43_syst = 0.9;                                  //low-ex network (efficiency)
+
+            double p44_syst = (ab39_syst + ab43_syst + ab52_syst + ab61_syst) / x43_syst;  //matching - generation demand(MWh / a)
+
+            //Renewables
+            double c7_renewables = o2_input;                        //available space 
+            double c8_renewables = 6.0;                               //radius
+            double c9_renewables = (Math.PI * Math.Sqrt(3) / 6 * c7_renewables) / (Math.PI * (Math.Pow(c8_renewables, 2)));   //maximum value
+            double c6_renewables = 1.0;                             //geo factor
+            double c10_renewables = c9_renewables * c6_renewables;  //anzahl sonden _ number of special??
+            double c13_renewables = 100.0;                            //lange der sonde _ length of the special??
```

Comment:
Oops, missed this! 
Not sure what makes most sense in this context, but 'sonde' could be probe or sample
anzahl sonden = Number of probes/samples
länge der sonde = Probe/sample length